src/java.base/share/classes/javax/crypto/CryptoPermissions.java

```diff
@@ -156,7 +155,7 @@ public void add(Permission permission) {
     }
 
     /**
-     * Checks if this object's PermissionCollection for permissons
+     * Checks if this object's PermissionCollection for permissions
```

Comment:
Just FYI and not for this review, but this class should really be updated to use proper javadoc comment style, which is to tag all objects with <code>PermissionCollection</code>

Reply:
No change required.